.golangci.yml

```diff
@@ -153,6 +153,12 @@ issues:
   # excluded by default patterns execute `golangci-lint run --help`
   exclude: []
 
+  exclude-rules:
+    # Exclude some linters from running on tests files.
+    - path: _test\.go$|^tests/|^samples/
+      linters:
+        - errcheck
```

Comment:
Oh man, I didn't know this option was available. Could you disable maligned on test as well please?

Reply:
Done